### PR TITLE
Fix CSS classes in BaseHtml

### PR DESCRIPTION
**Pull request by WinterSilence**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️
| New feature?  | ✔️
| Breaks BC?    | ✔️/❌
| Fixed issues  |  #19012

### CODE REVIEW

File: framework/helpers/BaseHtml.php [hunk 1]
```diff
@@ -1981,7 +1999,7 @@ public static function renderTagAttributes($attributes)
                     if (empty($value)) {
                         continue;
                     }
-                    $html .= " $name=\"" . static::encode(implode(' ', $value)) . '"';
+                    $html .= " $name=\"" . static::encode(implode(' ', static::splitCssClasses($classes))) . '"';
```
Comment (WinterSilence) [inline framework/helpers/BaseHtml.php:2002]: remove duplicate classes

**Comment by samdark** [comment]
Seems to fail tests...

**Comment by WinterSilence** [comment]
@samdark  no problem fix tests, I want discuss PR before do it.
Current implementation have incorrect behavior: `removeCssClass()`/ `addCssClass()` converts resulting array with string case to string

**Comment by WinterSilence** [comment]
@rob006 @Arhell what's you think about that changes?

**Pull request closed without merging**